src/CoreFx.Private.TestUtilities/tests/AssertExtensionTests.cs

```diff
@@ -0,0 +1,136 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Sdk;
+
+namespace CoreFx.Private.TestUtilities.Tests
+{
+    public class AssertExtensionTests
```

Comment:
Are these unit tests for the functionality of our test helpers? That shouldn't be necessary, and if it is then our helpers are too complex.